ChangeLog.md

```diff
@@ -11,6 +11,12 @@ Changes that affect Michelson code are tagged with `[CONTRACT]`.
 Unreleased
 ==========
 
+0.5.0
+=====
+* [#112](https://github.com/tz-wrapped/tezos-btc/pull/112)
+  - Update to use new version of multisig contract that also include chain id
+  in the signature payload.
```

Comment:
I would write explicitly that it's a sort of breaking change in sense that the new client won't work with previously deployed (old) multisig contracts.

Reply:
Updated